Sources/Controllers/2.0/OAPOIFilterViewController.mm

```diff
@@ -372,7 +372,10 @@ - (void)showEditCategoriesScreen
     OACustomPOIViewController *customPOIScreen = [[OACustomPOIViewController alloc] initWithFilter:_filter];
     customPOIScreen.delegate = self.delegate;
     customPOIScreen.refreshDelegate = self;
-    [self.navigationController pushViewController:customPOIScreen animated:YES];
+    UINavigationController *navController = [[UINavigationController alloc] initWithRootViewController:customPOIScreen];
```

Comment:
Why do you need to show OACustomPOIViewController fullscreen?

Reply:
One of the items in the task "Open fullscreen" for OACustomPOIViewController